packages/core/src/artifact/zip/ZipArtifactRepository.ts

```diff
@@ -0,0 +1,63 @@
+import type JSZip from 'jszip'
+import { minimatch } from 'minimatch/dist/mjs'
```

Comment:
racze sporadycznie jest potrzeba importowania z dist/, jeżeli zrobiłeś tak bo ts rzucał błędy to raczej kwestia braku typów, wtedy jesli biblioteka nie jest w TS to warto poszukać typów do niej -> https://www.npmjs.com/package/@types/minimatch